swift/looker/rtl/constants.swift

```diff
@@ -215,6 +219,37 @@ func asQ(_ value: Any?) -> String {
     return result
 }
 
+///
+/// OK with `try` that will throw an error
+/// Use with
+/// ```
+/// let me = try? oke(sdk.me()
+/// ```
+/// or
+/// ```
+/// do {
+///   let me = try oke(sdk.me()
+/// } catch {
+///   // handle error here
+/// }
+/// ```
+func okt<TSuccess, TError>(_ response: SDKResponse<TSuccess, TError>) throws -> TSuccess {
```

Comment:
Not the most familiar with this stuff, but it's weird to me that the `ok` functions between languages seem to behave differently. In TS, `ok` will possibly reject the promise, resulting in a throw, but in swift we're just swallowing the error?

I'd almost rather `ok` possibly throws in swift, and force all the call sites to, at the very least `try!` it.

Reply:
Agreed. Just had that conversation with @joeldodge79 and @dthorpe 

Let's see if we can get some swift feedback before I make that significant a change.